src/modules/dom.js

```diff
@@ -21,6 +22,7 @@ export function creatSubhtml(userName, userScore) {
   listelem.innerText = `${userName}: ${userScore}`;
   if (flag === false) {
     listelem.style.background = 'rgb(231 231 231)';
+    listelem.style.color = '#120212';
```

Comment:
- [OPTIONAL ] Please, It is always the best practice to fix all linter errors instead of disabling it  Linters are powerful and provide an easy way to scan your code for syntax errors that could lead to bugs. Or they can simply help keep a codebase clean, healthy, and consistent. When run, it will show all the issues and let you go through each one individually to fix them

https://github.com/Theophileaseh/Leaderboard/blob/54740ea2c22e432a6723a8f37158c52385e1dd91/src/modules/leaderScore.js#L5
https://github.com/Theophileaseh/Leaderboard/blob/54740ea2c22e432a6723a8f37158c52385e1dd91/src/modules/leaderScore.js#L12
https://github.com/Theophileaseh/Leaderboard/blob/54740ea2c22e432a6723a8f37158c52385e1dd91/src/modules/leaderScore.js#L18

https://github.com/Theophileaseh/Leaderboard/blob/54740ea2c22e432a6723a8f37158c52385e1dd91/src/modules/leaderScore.js#L18

https://github.com/Theophileaseh/Leaderboard/blob/54740ea2c22e432a6723a8f37158c52385e1dd91/src/index.js#L1